docs/lake/README.md

```diff
@@ -125,11 +125,10 @@ to either stdout, or to a new file in the chunk's directory.
 Here's an example using the same "count()" query as before:
 
 ```
-zed lake -f table map "count()"
+zed lake map -f text "count()"
```

Comment:
I opted to fix this one by going to `-f text` because `-f table` gives the ugly output:

```
$ zed lake map -f table "count()"
COUNT
275745
COUNT
276326
COUNT
290789
COUNT
619218
```

Depending on if `zed lake map` lives on in some form in the reworked `zed lake`, perhaps we could write something to emphasize the side effects of the per-file processing you get with `map`. But since this is so early in the text, I figure we might as well keep it simple.